L/libCEED/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libCEED"
+version = v"0.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/CEED/libCEED.git", "6134893e0067dc2a87bbbe22a551ae134ad5b885")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd libCEED
+sed -i "s/MEMCHK :=.*/MEMCHK := 0/g" Makefile 
+sed -i "s/CC_VENDOR :=.*/CC_VENDOR := gcc/g" Makefile 
+make -j$ncore
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf),
+    MacOS(:x86_64)
+]
```

Comment:
```suggestion
platforms = supported_platforms(exclude= x -> typeof(x) == Windows || typeof(x) == FreeBSD)
```
or use the new platform ctors, e.g. `Platform("x86_64", "linux"; libc="glibc")`.

Reply:
Thanks. I was a bit confused by the error message ```LoadError: ArgumentError: Platform `i686-linux-glibc` is not an officially supported platform```.

Reply:
It looks like the CI is still trying to build for `i686-w64-mingw32` and `x86_64-w64-mingw32`. Is there something I can do to fix this?